plugins/amazonq/codewhisperer/jetbrains-ultimate/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import software.aws.toolkits.gradle.intellij.IdeFlavor
+
+plugins {
+    id("toolkit-intellij-subplugin")
+}
+
+intellijToolkit {
+    ideFlavor.set(IdeFlavor.IU)
+}
+
+dependencies {
+    compileOnly(project(":plugin-amazonq:codewhisperer:jetbrains-community"))
```

Comment:
Should this also take in shared:jb-community or will it be transitive?

Reply:
transitive